x-pack/solutions/security/plugins/security_solution/public/configurations/page/configuration_tabs.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import { EuiTab, EuiTabs } from '@elastic/eui';
+import { useParams } from 'react-router-dom';
+import { SecurityPageName } from '@kbn/security-solution-navigation';
+import { CONFIGURATIONS_PATH } from '../../../common/constants';
+import { useNavigation } from '../../common/lib/kibana';
+
+import * as i18n from '../translations';
+export enum ConfigurationTabs {
+  integrations = 'integrations',
+  basicRules = 'basic_rules',
+  knowledgeSources = 'knowledge_sources',
+}
```

Comment:
Can we extract this to a `./constants.ts`? So all the files that need to import this enum don't have to load the component as well.

Reply:
Good point, done 👍